DrivingChassis.cpp

```diff
@@ -87,8 +233,60 @@ void DrivingChassis::driveForward(float mmDistanceFromCurrent, int msDuration) {
  *  @note pidmotorInstance->overrideCurrentPosition(0); can be used to "zero out" the motor to
  * 		  allow for relative moves. Otherwise the motor is always in ABSOLUTE mode
  */
-void DrivingChassis::turnDegrees(float degreesToRotateBase, int msDuration) {
+void DrivingChassis::turnDegreesFromInterpolation(float degreesToRotateBase, int msDuration) {
+	   myleft -> overrideCurrentPosition(0);
+	   myright -> overrideCurrentPosition(0);
+	   myleft -> startInterpolationDegrees(degreesToRotateBase * WHEEL_DEGREES_TO_BODY_DEGREES, msDuration, LIN);
+	   myright -> startInterpolationDegrees(degreesToRotateBase * WHEEL_DEGREES_TO_BODY_DEGREES, msDuration, LIN);
+}
+
+DrivingStatus DrivingChassis::turnToHeading(float degreesToRotateBase, int msDuration){
```

Comment:
again, seperate this into 2 functions, one to start the move, another to check on completion

Reply:
same as above